src/cli.rs

```diff
@@ -255,6 +269,22 @@ mod cli_tests {
         assert_eq!(parameters.output_format_or_default(), super::DEFAULT_FORMAT);
     }
 
+    #[test]
+    fn encoding_resolution() {
```

Comment:
lets add a test that defines the behavior if I ask for an unknown decoding via the cli

Reply:
Done